tests/rados/test_libcephsqlite.py

```diff
@@ -0,0 +1,195 @@
+"""
+Module to verify libcephsqlite's ability to reopen a database connection if
+current connection is down/blocklisted.
+"""
+import random
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from tests.rados.monitor_configurations import MonConfigMethods
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    # CEPH-83583664
+    Covers:
+        - BZ-2130867
```

Comment:
Add details of polarion test coverage in bugzilla

Reply:
Will it be a comment added to the bug, or is there a separate field to be updated?

Reply:
Add qe_test_coverage flag to "?" and in the linked section add polarion link. If test is automated then the flag to be changed to "+"